.vsts-ci.yml

```diff
@@ -129,11 +129,13 @@ jobs:
         TASK: bdist
         PYTHON_VERSION: 3.5
   steps:
-  - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
-    displayName: Enable conda
+  - powershell: |
+      Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+      Write-Host "##vso[task.setvariable variable=AZURE]true"
```

Comment:
I think we should have the same for Appveyor. I mean, `$env:APPVEYOR = "true"`. And break the condition `$env:TASK -eq "appveyor-python"` into `$env:APPVEYOR -eq "true"` and `$env:TASK -eq "python"`.

Reply:
good idea,  ok!

Reply:
let me know if this is what you're thinking: https://github.com/microsoft/LightGBM/pull/2962/commits/79c48abcf9ad0ae2c843b033e99587562ea90687

I thought it was better to set `APPVEYOR` inside the script since that is a variable already provided on AppVeyor builds. Setting in the scripts guarantees that we're overwriting it. It also means we don't have to copy it `n_tasks` times in the `matrix:` block

Reply:
oops, forgot the task name: https://github.com/microsoft/LightGBM/pull/2962/commits/a331e2b1a13da7a9b82ae2d37b2677499d2bd8af